assets/js/modules/analytics-4/datastore/__fixtures__/available-audiences.json

```
@@ -0,0 +1,38 @@
+[
+    {
+        "name": "properties/12345/audiences/1",
+        "description": "All users",
+        "displayName": "All Users",
+        "audienceType": "DEFAULT_AUDIENCE",
+        "audienceSlug": "all-users"
+    },
+    {
+        "name": "properties/12345/audiences/2",
+        "description": "Users who have made a purchase",
+        "displayName": "Purchasers",
+        "audienceType": "DEFAULT_AUDIENCE",
+        "audienceSlug": "purchasers"
+    },
+    {
+        "name": "properties/12345/audiences/3",
+        "description": "People who visited the site for the first time",
+        "displayName": "New visitors",
+        "audienceType": "SITE_KIT_AUDIENCE",
+        "audienceSlug": "new-visitors"
+    },
+    {
+        "name": "properties/12345/audiences/4",
+        "adsPersonalizationEnabled": true,
+        "description": "People who have visited your site at least once before",
+        "displayName": "Returning visitors",
+        "audienceType": "SITE_KIT_AUDIENCE",
+        "audienceSlug": "returning-visitors"
+    },
+    {
+        "name": "properties/12345/audiences/5",
+        "description": "Description",
+        "displayName": "Test Audience",
+        "audienceType": "USER_AUDIENCE",
+        "audienceSlug": "test-users"
```

Hey @ankitrox just a quick heads up - please note the `audienceSlug` is set to an empty string for user-defined audiences, as can be seen here:

https://github.com/google/site-kit-wp/blob/109d7505a594b2d8db4cb499c14e4abb98b92a8a/includes/Modules/Analytics_4.php#L2244-L2245

```suggestion
        "audienceSlug": ""
```